diff --git a/node_modules/@meshsdk/react/dist/hooks/useWallet.d.ts b/node_modules/@meshsdk/react/dist/hooks/useWallet.d.ts
index 79b1427..8aec6b8 100644
--- a/node_modules/@meshsdk/react/dist/hooks/useWallet.d.ts
+++ b/node_modules/@meshsdk/react/dist/hooks/useWallet.d.ts
@@ -2,7 +2,7 @@ export declare const useWallet: () => {
     name: string;
     connecting: boolean;
     connected: boolean;
-    wallet: BrowserWallet;
+    wallet: import("@meshsdk/core").BrowserWallet;
     connect: (walletName: string) => Promise<void>;
     disconnect: () => void;
     error: unknown;
